digital_comms/ccam.py

```diff
@@ -98,13 +160,35 @@ def population_density(self):
             return self.population / total_area
 
     def add_pcd_sector(self, pcd_sector):
+        """Add a postcode sector to the local area district.
+
+        Arguments
+        ---------
+        pcd_sector: PostcodeSector
+            Representation of a postcode sector that needs to be
+            added to the local area district
+        """
         self._pcd_sectors[pcd_sector.id] = pcd_sector
 
     def add_asset(self, asset):
+        """Add an asset to postcode sector to the local area district
```

Comment:
Does this not just add an asset to the pcd_sector, and the pcd_sector is added to the LAD in the above method?

Hence, should this not just be """Add an asset to postcode sector"""?

Reply:
Done